dataland-frontend/src/router/index.ts

```diff
@@ -22,10 +24,12 @@ const routes = [
   {
     path: "/",
     name: "Welcome to Dataland",
-    component: WelcomeDataland,
-    props: {
-      isMobile: /Android|webOS|iPhone|iPad|iPod|BlackBerry|Windows Phone|IEMobile|OperaMini/i.test(navigator.userAgent),
-    },
```

Comment:
We used to have this in order to show mobile users a page that says "mobile version will be ready soon".
Is this gone now?  Or do we just have a different mechanism for it?

Reply:
I am waiting for a feedback for this point. For my understanding it should be possible to login and register from mobile but let's wait for the feedback

Reply:
Ah ok.  That really surprises me, because it currently does not really make sense to login via mobile device, because the whole Dataland page is not really usable on a small screen currently.

On dataland.com you can currently see what happens if you log in via mobile device. You kind of get a "note" that Dataland is currently not suitable for mobile devices.
@fsoffel and I think it is ok to keep that page, until (in another ticket) a proper mobie page is set up.